agent/agents/mysql/slowlog/parser/parser.go

```diff
@@ -219,37 +221,50 @@ func (p *SlowLogParser) parseHeader(line string) {
 			p.event.Db = submatch[1]
 		}
 
-		m := metricsRe.FindAllStringSubmatch(line, -1)
-		for _, smv := range m {
+		if !strings.Contains(line, ":") {
+			return
+		}
+
+		line = strings.Replace(line, ": ", ":", -1)
+		for _, kv := range strings.Split(line, " ") {
+			if len(kv) == 0 {
+				continue
+			}
+			kv2 := strings.Split(kv, ":")
+			k := strings.TrimSpace(kv2[0])
+			v := ""
+			if len(kv2) > 1 {
+				v = strings.TrimSpace(kv2[1])
+			}
```

Comment:
Since we're replacing an existing regex funcs, it'd be a good idea to:
1. Extract logic in a separate func
2. Test new logic in comparison to the func being replaced

Reply:
What do u mean?

Reply:
I have provided a benchmark for old and new version of this parser.

Reply:
1. Create a func to put in right in place of `metricsRe.FindAllStringSubmatch(line, -1)`
2. Add unit tests